src/InitialConditions/initial_conditions.jl

```diff
@@ -115,6 +115,42 @@ function (initial_condition::AgnesiHProfile)(params)
     return local_state
 end
 
+"""
+    ScharProfile(; perturb = false)
+An `InitialCondition` with a prescribed Brunt-Vaisala Frequency
+"""
+Base.@kwdef struct ScharProfile <: InitialCondition end
+
+function (initial_condition::ScharProfile)(params)
+    function local_state(local_geometry)
+        FT = eltype(params)
+
+        thermo_params = CAP.thermodynamics_params(params)
+        g = CAP.grav(params)
+        R_d = CAP.R_d(params)
+        cp_d = CAP.cp_d(params)
+        cv_d = CAP.cv_d(params)
+        p₀ = CAP.MSLP(params)
+        (; x, z) = local_geometry.coordinates
+        θ₀ = FT(280.0)
+        𝒩 = FT(0.01)
```

Comment:
I would prefer naming it N or something like buoy_freq (is that what it is?), as I don't think the unicode 𝒩 helps much here. Up to you though.

Reply:
Noted, this choice was mostly to follow existing literature examples; I can change this to something more verbose. Testing unicode out in REPL - the rendering can be unclear if it has complex characters trailing it (or leading up to it).